Interface/Harp.CameraController/Harp.CameraController.csproj

```diff
@@ -2,25 +2,34 @@
 
   <PropertyGroup>
     <Title>Harp.CameraController</Title>
-    <Authors></Authors>
-    <Copyright>Copyright ©  2023</Copyright>
+    <Authors>harp-tech</Authors>
+    <Copyright>Copyright © harp-tech and Contributors 2023</Copyright>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <GeneratePackageOnBuild Condition="'$(Configuration)'=='Release'">true</GeneratePackageOnBuild>
     <Description>Bonsai Library containing interfaces for data acquisition and control of Harp CameraController devices.</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>Harp CameraController Bonsai Rx</PackageTags>
-    <PackageProjectUrl></PackageProjectUrl>
-    <PackageLicenseExpression></PackageLicenseExpression>
-    <PackageIcon></PackageIcon>
-    <PackageOutputPath></PackageOutputPath>
+	<PackageProjectUrl>https://harp-tech.org</PackageProjectUrl>
```

Comment:
There are some weird added spaces in the csproj. Can you run auto format on it?

Reply:
Weird... Done.